client/signup/steps/import-url/index.jsx

```diff
@@ -289,7 +285,7 @@ class ImportURLStepComponent extends Component {
 
 		const headerText = translate( 'Where can we find your old site?' );
 		const subHeaderText = translate(
-			'Enter your Wix or GoDaddy GoCentral site URL, sometimes called a domain name or site address.'
+			'Enter your Wix site URL, sometimes called a domain name or site address.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Enter your Wix site's URL, sometimes called a domain name or site address.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-godaddy-gocentral-importer)

Reply:
Let's do this